lib/jekyll-admin/static_file.rb

```diff
@@ -0,0 +1,61 @@
+module JekyllAdmin
+  class StaticFile
```

Comment:
How does this relate to https://github.com/jekyll/jekyll/blob/master/lib/jekyll/static_file.rb? Could we create a mixin and monkey-patch it like we do with the other content types? https://github.com/jekyll/jekyll-admin/blob/master/lib/jekyll-admin.rb#L32-L37